src/mongo/models.ts

```diff
@@ -0,0 +1,28 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+const EVENT_TRACKER = 'EventTracker';
```

Comment:
should stick to the same convention, either camelCase or TitleCase for collection name